src/api/flaskr/service/shifu/dtos.py

```
@@ -681,3 +689,24 @@ def __json__(self):
+        self.bid = bid
+        self.children = children
+
+    def __json__(self):
+        return {
+            "bid": self.bid,
+            "children": self.children,
+        }
+
+
+@register_schema_to_swagger
+class ReorderOutlineDto:
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)